src/safeds/data/image/containers/_image.py

```diff
@@ -305,6 +305,49 @@ def adjust_brightness(self, factor: float) -> Image:
         image_copy._image = ImageEnhance.Brightness(image_copy._image).enhance(factor)
         return image_copy
 
+    def add_gaussian_noise(self, standard_deviation: float, opacity: float = 1.0) -> Image:
+        if standard_deviation < 0:
+            raise ValueError("Standard deviation has to be 0 or bigger.")
+
+        if opacity < 0 or opacity > 1:
+            raise ValueError("Opacity has to be between 0 and 1.")
+
+        if opacity == 0:
+            warnings.warn(
+                "Opacity is 0, this will not make changes to the image.",
+                UserWarning,
+                stacklevel=2,
+            )
+
+        image_copy = copy.deepcopy(self)
+        img_noise = PIL.Image.effect_noise((image_copy.width, image_copy.height), standard_deviation)
+        image_copy._image = PIL.Image.blend(image_copy._image.convert("L"), img_noise.convert("L"), 0.5)
+        return image_copy
+
+    def blend(self, other: Image, alpha: float = 0.5) -> Image:
```

Comment:
I don't like `alpha` here:
1. For `add_gaussian_noise` we control opacity, here we control transparency.
2. It's not entirely clear whether it affects the transparency of the original image or the transparency of the other image.

I'd suggest to
1. Configure the opacity here for the sake of consistency (rename `alpha` to `opacity`)
2. Rename the method to something like `add_image` or `overlay_image`. I think `add_image` might be the best choice since it's consistent with `add_gaussian_noise` and because there are other ways to combine images, like subtracting one or multiplying them.

Reply:
Actually, scratch that. I don't see a major use-case for a method that interpolates between two  images anyway. Please remove the method.